plugins/home/src/components/CustomHomepage/types.ts

```diff
@@ -98,6 +98,14 @@ export type CustomHomepageGridProps = {
    * @defaultValue false
    */
   preventCollision?: boolean;
+
+  /**
+   * Hook that overwrite default useHomeStorage hook. For example, use database to storage homepage layout data
```

Comment:
can this be accomplished by instead implementing a custom version of the storage api?

Reply:
Yes, that's the intended way.

Follow along with these instructions, and then everything that uses the storage API will have it persisted across sessions.

https://github.com/backstage/backstage/tree/master/plugins/user-settings-backend